InstallSpinnaker.sh

```diff
@@ -680,6 +680,8 @@ install_spinnaker
 
 # Touch a file to tell other scripts we installed Cassandra.
 touch /opt/spinnaker/cassandra/SPINNAKER_INSTALLED_CASSANDRA
```

Comment:
Is this still necessary (are there other checks relying on this) because below you deleted one.